pandas/core/tools/datetimes.py

```diff
@@ -701,8 +701,15 @@ def to_datetime(
         Specify a date parse order if `arg` is str or its list-likes.
         If True, parses dates with the day first, eg 10/11/12 is parsed as
         2012-11-10.
-        Warning: dayfirst=True is not strict, but will prefer to parse
-        with day first (this is a known bug, based on dateutil behavior).
+
+        .. warning::
+
+            dayfirst=True is not strict, but will prefer to parse
+            with day first (this is a known bug, based on dateutil behavior).
```

Comment:
i wouldn't say this is a known bug unless you can point to an authoritative reference.

Reply:
I didn't add this, it's from https://github.com/pandas-dev/pandas/pull/7599 , I just put it into a warning block